docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -46,46 +46,61 @@ You can find the solution in the `security-jpa-quickstart` link:{quickstarts-tre
 :sectnums:
 :sectnumlevels: 3
 
-== Create a Maven project
+== Create and verify the Maven project
 
 For Quarkus Security to be able to map your security source to Jakarta Persistence entities, ensure that the Maven project that is used in this tutorial includes the `security-jpa` or the `security-jpa-reactive` extension.
+
+[NOTE]
+====
+xref:hibernate-orm-panache.adoc[Hibernate ORM with Panache] is used to store your user identities but you can also use xref:hibernate-orm.adoc[Hibernate ORM] with the `security-jpa` extension.
+Both xref:hibernate-reactive.adoc[Hibernate Reactive] and xref:hibernate-reactive-panache.adoc[Hibernate Reactive with Panache] can be used with the `security-jpa-reactive` extension.
+
+You must also add your preferred database connector library.
+The instructions in this example tutorial use a PostgreSQL database for the identity store.
+====
+
 You can either create a new Maven project with the Security Jakarta Persistence extension or you can add the extension to an existing Maven project.
 
-* To create the Maven project with Hibernate ORM, use the following command:
+=== Create the Maven project
+
+.Create a new Maven project with the Jakarta Persistence extension
```

Comment:
.Create a new Maven project with the Jakarta Persistence extension -> . Create a new Maven project with the Jakarta Persistence extension

If those should be numbered steps in the end.